odl/contrib/shearlab/shearlab_operator.py

```diff
@@ -171,8 +179,51 @@ def _call(self, x):
 
             @property
             def adjoint(self):
-                """The adjoint operator."""
-                return ShearlabOperatorAdjointInverse()
+                """The inverse operator."""
+                op = self
+
+                class ShearlabOperatorInverseAdjoint(odl.Operator):
+
+                    """
+                    Adjoint of the inverse/Inverse of the adjoint
+                    of shearlet transform.
+
+                    See Also
+                    --------
+                    odl.contrib.shearlab.ShearlabOperator
+                    """
+
+                    def __init__(self):
+                        """Initialize a new instance.
+                        Parameters
+                        ----------
+                        op : `ShearlabOperator`
+                              The operator which this should be the
+                              inverse of the adjoint of.
```

Comment:
Remove the Parameters section

Reply:
As pointed out in #1358, this also goes for lines 78, 117, and 164 (although these lines are not changed in these commits).